lib/hooks/queries/useInfiniteMarkets.ts

```diff
@@ -39,6 +39,10 @@ export type QueryMarketData = Market<IndexerContext> & {
   prediction: { name: string; price: number };
 };
 
+const WHITELISTED_TRUSTED_CREATORS = [
+  "dE2cVL9QAgh3MZEK3ZhPG5S2YSqZET8V1Qa36epaU4pQG4pd8",
```

Comment:
What's this address? Could we use env variable for it?

Reply:
Done ✅